### PR TITLE
The pondering has begun

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
image

## Why It's Good For The Game
FFerBD5XoAMvcgO

## Changelog

:cl:
add: Added Pondering Orb (It ponders)
/:cl:

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -6462,8 +6462,8 @@
 /obj/machinery/door/firedoor,
 /obj/structure/window/reinforced,
 /obj/machinery/computer/cargo/request{
-	icon_state = "computer";
```
Comment (JohnFulpWillard) [inline _maps/map_files/MetaStation/MetaStation.dmm:6465]: :face_with_raised_eyebrow:

File: code/game/objects/items/pondering_orb.dm [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/obj/item/pondering_orb
+	name = "Pondering Orb"
+	icon = 'icons/obj/pondering_orb.dmi'
+	icon_state = "pondering_orb"
+	desc = "Born to ponder."
+
+/obj/item/pondering_orb/attack_hand(mob/user, list/modifiers)
+	balloon_alert(user, "You start pondering on the orb...")
```
Comment (ATH1909) [inline code/game/objects/items/pondering_orb.dm:8]:
```suggestion
	balloon_alert(user, "You start pondering the orb...")
```

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (Ghilker) [reply _maps/map_files/MetaStation/MetaStation.dmm:6465]: i didn't do that

**Comment by optimumtact** [comment]
we're in a freeze bro

**Comment by esainane** [comment]
that's cold man

**Comment by optimumtact** [comment]
I did him the dirty

**Comment by LemonInTheDark** [comment]
:sadkirby:

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (LemonInTheDark) [reply _maps/map_files/MetaStation/MetaStation.dmm:6465]: ~~did you use fastdmm?~~

**Pull request closed without merging**